### PR TITLE
Update ShivangShandilya.json

### PR DESCRIPTION
**Pull request by ShivangShandilya**
## Changes proposed

This PR updates  ShivangShandilya.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4322"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

File: data/ShivangShandilya.json [hunk 1]
```diff
@@ -2,73 +2,119 @@
...
       "url": "https://www.linkedin.com/in/shivang-shandilya-8b926b218/",
       "icon": "FaLinkedin"
     },
     {
-      "name": "Follow me on GitHub",
-      "url": "https://github.com/ShivangShandilya",
-      "icon": "FaGithub"
+      "group": "Socials",
+      "name": "Instagram - Just Connect, why thinking?",
+      "url": "https://www.instagram.com/pradumnasaraf",
```
Comment (SaraJaoude) [inline data/ShivangShandilya.json:36]: Please check the link to your instagram as this does not appear to be your account

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
more details in inline comment

File: data/ShivangShandilya.json (hunk 1 above)
Comment (ShivangShandilya) [reply data/ShivangShandilya.json]: Oops, I was using Pradumna's Linkfree file as a reference looks like I forgot to change the links. Thanks for pointing this out 😁😁

Comment (ShivangShandilya) [reply data/ShivangShandilya.json]: Issue has been resolved, do tell me if there is anything else that needs to be corrected 😁

**Review by SaraJaoude — APPROVED** [review]

**Pull request merged**